less/navigation.less

```diff
@@ -1 +1,55 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+.navigation {
+
+  @media @phone {
+    max-width: 500px;
+    width: 100%;
+    margin: 0 auto;
+
+  }
+
+    width: 100%;
+    height: 80px;
+    background-color: #f9f9f9;
+    display: flex;
+    justify-content: center;
```

Comment:
You could have a mixin for display:flex, justify-content: center and add them on every content that you use these attributes on.